### ISSUE TITLE
build fails due to wrong types in host.Users() api call

### ISSUE THREAD

**Issue by ranjib** [shirou/gopsutil#87]
host.Users() uses acrh specific utmp struct types. In raspberry pi this returns byte array straight instead of int8 array for u.User field. 

go version : 1.5.1
OS: Linux, Ubuntu 15.05,  
Arch: armhf (Raspberry Pi 2)

```sh
../../shirou/gopsutil/host/host_linux.go:110: cannot use u.User[:] (type []byte) as type []int8 in argument to common.IntToString
../../shirou/gopsutil/host/host_linux.go:111: cannot use u.Line[:] (type []byte) as type []int8 in argument to common.IntToString
../../shirou/gopsutil/host/host_linux.go:112: cannot use u.Host[:] (type []byte) as type []int8 in argument to common.IntToString
```

**Comment by shirou** [comment]
I fixed at 3303647209557312e5db51450ea8bbdef56d5176. and tested scaleway https://www.scaleway.com/ go 1.5.1, Ubuntu 15.04
Coud you test on your Raspberry Pi 2?

Thank you.

**Comment by ranjib** [comment]
awesome :-) . thanks for the quick fix
